docs/ui.md

```diff
@@ -4,7 +4,7 @@ title: Building the Connection Dialog
 
 ## The Connection Dialog
 
-The Connection Dialog prompts the user to enter connection and authentication information that will passed into the connector builder script to build the connection string. The dialog appears when creating a new connection or editing an existing connection on both Tableau Desktop and Server. 
+The Connection Dialog prompts the user to enter connection and authentication information that will passed into the connector builder script to build the connection string. The dialog appears when creating a new connection or editing an existing connection on both Tableau Desktop and Server.
```

Comment:
Couple of minor tweaks:
change "...authentication information that will passed..."  to "...authentication information that is passed into the..."
add Tableau: "...on both Tableau Desktop and Tableau Server."